### DISCUSSION TITLE
Add column sorting

### DISCUSSION THREAD

**Discussion by shogd3838** [getseabird/seabird#55] [Ideas]
Hello I don't know if it was already asked or if it's planned in future release but the ability to sort pods or any other content of the cluster  by clicking on the column we want to order by would be very helpful.
For example I wanted to sort all the pods from my cluster by their status (ready, not ready), or by their CPU or Memory usage.

**Comment by jgillich** [comment]
Basic sorting has been added in 19d1d897a6c1e3f8601959b7644003955e360254

Not all columns are sortable yet, that will be extended over time
